free-courses-en.md

```diff
@@ -79,11 +79,13 @@
 
 ### Clojure
 
+* [BraveClojure](http://www.braveclojure.com)
```

Comment:
Yep, perfect. Please use this one @MathewtheCoder, but in `free-programming-books.md` instead of here. :)